nixos/modules/services/networking/kea.nix

```diff
@@ -254,9 +254,10 @@ in
       DynamicUser = true;
       User = "kea";
       ConfigurationDirectory = "kea";
-      RuntimeDirectory = "kea";
       StateDirectory = "kea";
       UMask = "0077";
+    } // lib.optionalAttrs (lib.versionOlder config.system.stateVersion "23.11") {
+      RuntimeDirectory = "kea";
```

Comment:
If this is indeed a bug, we should not tie this to the stateVersion and fix it for everyone.